internal/repository/users.go

```diff
@@ -0,0 +1,122 @@
+package repository
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/zhuravlev-pe/course-watch/internal/core"
+	"github.com/zhuravlev-pe/course-watch/pkg/security"
+)
+
+const (
```

Comment:
I don't really see a reason to inject the table name via a constant. All remaining identifiers are hardcoded in Sprintf(). We are not using anything fancy (e.g. Squirrel) to build queries. The only effect that I see is that Goland's query analyzer goes bananas over this :) I suggest to get rid of this constant and Sprintf() unless there is a good reason to keep this